src/preprocessing/hlc.py

```
@@ -0,0 +1,64 @@
+from typing import Tuple
```

Ruff

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)